bootloader/lkenv/lkenv_test.go

```diff
@@ -59,6 +60,7 @@ var (
 )
 
 func (l *lkenvTestSuite) SetUpTest(c *C) {
+	l.BaseTest.SetUpTest(c) //Just to ensure SNAPD_DEBUG unset at the moment
```

Comment:
(nitpick) I personally would remove the comment, it will become stale at some point. Yes, we originally added this for SNAPD_DEBUG but it offers more benefits like the `AddCleanup` helper and once this gets used it will no longer be accurate. Given that we have a test that ensure that removing it will trigger an error I think it's ok to remove the comment. Same for the other instance where this is added.

Fwiw, if we want to keep a comment I think it needs to be clearer, I'm not sure my future self in 3 month will understand what this means. I.e. why do we care that SNAPD_DEBUG is set or unset is not visible from the comment.